vcluster/manage/backup-restore.mdx

```diff
@@ -1,175 +1,243 @@
 ```
 
-When creating a new vCluster, you can also create the vCluster from a snapshot via:
+:::tip
+You can also use snapshots to clone a vCluster or migrate it to a new cluster.
+:::
+
+When creating a new vCluster, you can restore it directly from a snapshot:
+
 ```bash
-# restore from an OCI snapshot (use local credentials)
+# Restore a new vCluster from an OCI snapshot (uses local credentials).
 vcluster create my-vcluster --restore oci://ghcr.io/my-user/my-repo:my-tag
 
-# migrate an existing vCluster by restoring from a snapshot and applying a new vcluster.yaml
+# Migrate an existing vCluster by restoring from a snapshot and applying a new vcluster.yaml.
 vcluster create my-vcluster --upgrade -f vcluster.yaml --restore oci://ghcr.io/my-user/my-repo:my-tag
 ```
 
-If a restore should fail, the following happens:
-* If the restore was used with `vcluster create` the vCluster is deleted.
-* If the restore was used with `vcluster restore` the restore is aborted. Please make sure to retry the restore until it succeeds or otherwise the vCluster might end up in a broken state.
+If a restore fails:
+- When using `vcluster create`, the vCluster is automatically deleted.
+- When using `vcluster restore`, the restore process is aborted. You must retry the restore to avoid leaving the vCluster in an inconsistent or broken state.
 
-:::tip
-Creating a vCluster from a snapshot can also be an effective method to clone a vCluster or migrate a vCluster to a new cluster.
-:::
 
 ### Limitations
 
-There are a couple of limitations when using the vCluster CLI to backup and restore a vCluster:
+The vCluster CLI has the following limitations when backing up and restoring a vCluster:
 
-* The vCluster CLI does not support backing up persistent volumes or persistent volume claims.
-* Snapshotting a k0s vCluster only works with using the `pod-exec` flag.
-* Restoring a vCluster with the k0s distro is not supported.
-* When restoring a vCluster with the external database backend, vCluster doesn't truncate the external database before restoring the snapshot.
-* When restoring a vCluster within a different namespace or with a different name than the original vCluster, the vCluster certificates are changed. This is expected behavior to avoid having multiple vCluster use the same certificates.
+- Backing up persistent volumes (PVs) or persistent volume claims (PVCs) is not supported.
+- Snapshotting a sleeping vCluster is not supported.
+- Snapshotting a k0s-based vCluster requires the `--pod-exec` flag.
+- Restoring a vCluster running the k0s distribution is not supported.
+- When restoring a vCluster that uses an external database backend, the database is not truncated before the snapshot is restored. Manual intervention is required.
+- Restoring a vCluster to a different namespace or with a different name changes the vCluster certificates. This prevents multiple vClusters from sharing the same certificates, which is expected behavior.
```

Comment:
We cannot use plular `vclusters`, this needs to be changed to `virtual clusters`. This is how marketing messaging around this:

> Never refer to a virtual cluster as a “vCluster” or “vClusters”. vCluster is a product that helps you create and manage virtual clusters.